sdk/authentication.ts

```diff
@@ -25,6 +25,11 @@ class Authentication {
 
     if (scopes.includes(JWTScope.Embed) && embed) {
       claims['embed'] = snakeCaseKeys(embed)
```

Comment:
@paul-gr4vy there's a better way to do this. See https://github.com/bendrucker/snakecase-keys#exclude